tests/tasks/secrets/test_vault_secret.py

```diff
@@ -0,0 +1,147 @@
+import os
+import hvac
```

Comment:
You'll want to use `pytest.importorskip` here so these tests are skipped if this module is not available

Reply:
added purest.importorskip("hvac") to __init__.py